### PR TITLE
Issues

### PR DESCRIPTION
**Pull request by grogou**

### CODE REVIEW

**Review by arsendovlatyan — COMMENTED** [review]
@eriktad why we need `slash_url` method here?

File: core/admin_menu/class-btm-admin-task-page-table.php [hunk 1]
```diff
@@ -70,68 +70,77 @@ public function on_hook_page_render_tasks(){
...
-			if ( ! empty( $_REQUEST['_wp_http_referer'] ) ) {
-				wp_redirect( remove_query_arg( array( '_wp_http_referer', '_wpnonce', 'action', static::BULK_ACTION_DELETE, ), wp_unslash( $_SERVER['REQUEST_URI'] ) ) );
-				exit;
-			}
-		} else {
-			if ( ! empty( $_REQUEST['_wp_http_referer'] ) ) {
-				wp_redirect( remove_query_arg( array( '_wp_http_referer', '_wpnonce' ), wp_unslash( $_SERVER['REQUEST_URI'] ) ) );
-				exit;
-			}
+	private function slash_url(){
```
Comment (arsendovlatyan) [inline core/admin_menu/class-btm-admin-task-page-table.php]: @eriktad why we need this method ?

File: core/admin_menu/class-btm-admin-page-task-single.php [hunk 2]
```diff
@@ -56,9 +56,11 @@ public function on_hook_page_render_task_single(){
 			}
 		}
 		$dao = BTM_Task_Single_View_Dao::get_instance();
-		$data = $dao->get_task( $task_id );
-		$data = $data[0];
-		$task_status = $data->get_status()->get_value();
+		$task = $dao->get_task( $task_id );
+		if( $task === false ){
+			throw new InvalidArgumentException( 'Wrong task id - ' . $task_id );
+        }
```
Comment (arsendovlatyan) [inline core/admin_menu/class-btm-admin-page-task-single.php:62]: @eriktad fix spacing.

File: core/admin_menu/class-btm-admin-task-page-table.php (hunk 1 above)
Comment (grogou) [reply core/admin_menu/class-btm-admin-task-page-table.php]: I remove this method... Now we don't need this functionality

File: core/admin_menu/class-btm-admin-page-task-single.php (hunk 2 above)
Comment (grogou) [reply core/admin_menu/class-btm-admin-page-task-single.php:62]: I think it is GitHub code formatting because in my local file all ok!...

**Review by arsendovlatyan — APPROVED** [review]
Looks good, thanks @eriktad !

**Pull request merged**